hudi-cli/src/main/java/org/apache/hudi/cli/commands/RepairsCommand.java

```diff
@@ -77,7 +77,9 @@ public String deduplicate(
           help = "Spark executor memory") final String sparkMemory,
       @CliOption(key = {"dryrun"},
           help = "Should we actually remove duplicates or just run and store result to repairedOutputPath",
-          unspecifiedDefaultValue = "true") final boolean dryRun)
+          unspecifiedDefaultValue = "true") final boolean dryRun,
+      @CliOption(key = {"dedupeType"}, help = "Check DeDupeType.scala for valid values",
+          unspecifiedDefaultValue = "insertType") final String dedupeType)
```

Comment:
It's better to show the three types in help string and have a type check at first line of command.

Reply:
Done.